common/models/user.js

```diff
@@ -812,6 +866,22 @@ module.exports = function(User) {
       }
     );
 
+    UserModel.remoteMethod(
+      'changePassword',
+      {
+        description: 'Change password of the current user.',
+        accepts: [
+          {arg: 'id', type: 'any',
+            http: ctx => ctx.req.accessToken && ctx.req.accessToken.userId,
+          },
+          {arg: 'oldPassword', type: 'string', required: true, http: {source: 'form'}},
+          {arg: 'newPassword', type: 'string', required: true, http: {source: 'form'}},
+          {arg: 'options', type: 'object', http: 'optionsFromRequest'},
+        ],
+        http: {verb: 'POST', path: '/change-password'},
```

Comment:
We typically use `POST` for non-RESTful actions, see e.g. `login`, `logout` or `resetPassword`.